### ISSUE TITLE
FCN head classes doesn't include background in COCO

### ISSUE THREAD

**Issue by LouieYang** [uber-research/UPSNet#55]
I noticed that for fcn head, the number of class logits is 133 for COCO and doesn't include background (I mean black area in annotation image). I am wondering if you have marked the bg in annotation images to 255 so that it is ignored in CE loss or is there anything I missed.

Many thanks

**Comment by YuwenXiong** [comment]
background is set to 255 and is ignored.

**Comment by qq237942920** [comment]
@YuwenXiong @LouieYang I found that the folder `panoptic_train2017_semantic_trainid_stff` set background (in coco gt) to 0 instead of 255, or is there anything I missed?

**Comment by YuwenXiong** [comment]
@qq237942920 @LouieYang I just found that the code I provided had a problem, it would set background to 0 instead of 255. Now it has been fixed, please pull the latest code and rerun `bash init_coco.sh` to regenerate correct coco panoptic gt

**Comment by discretecoder** [comment]
I am training on my own dataset with the same setting as cityscapes i.e same number of data (different resolution of images), city images so similar classes. the RCNN results are quite good but the FCN- semantic results are really bad especially for dense city images. is there any solution to improve the performance that you could suggest?
